datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -21,7 +21,6 @@
 # match integration's version within the __about__.py module
 VERSION = re.compile(r'__version__ *= *(?:[\'"])(.+?)(?:[\'"])')
 DOGWEB_JSON_DASHBOARDS = (
-    'activemq',
```

Comment:
Let's do this in a separate PR periodically rather than the ddev changelog showing similar changelog lines for each integration